csharp/src/Google.Protobuf/Google.Protobuf.csproj

```diff
@@ -6,7 +6,7 @@
     <AssemblyTitle>Google Protocol Buffers</AssemblyTitle>
     <VersionPrefix>3.6.0</VersionPrefix>
     <Authors>Google Inc.</Authors>
-    <TargetFrameworks>netstandard1.0;net45</TargetFrameworks>
```

Comment:
Fixed in https://github.com/jtattermusch/protobuf/pull/9.   You can rebase and the diff should disappear from this this PR (btw, I think fixes like this should be in a separate commit - then it's easier to remove then once it has been fixed in the target branch).

Reply:
Done